lib/matplotlib/__init__.py

```diff
@@ -244,7 +244,7 @@ def _set_logger_verbose_level(level_str='silent', file_str='sys.stdout'):
                 fileo = open(file_str, 'w')
                 # if this fails, we will just write to stdout
             except IOError:
-                warnings.warn('could not open log file "{0}"'
+                logging.warning('could not open log file "{0}"'
```

Comment:
throughout the PR:
You should use the module-level logger
`_log = logging.getLogger(__name__)` then `_log.warning(...)`.

Reply:
- Ah, right. I even read that I should do that, but for some reason it didn't register.
- Thanks for pointing that out. I wasn't really paying attention to that.